lib/functions-filters.php

```diff
@@ -2,9 +2,29 @@
 
 // Custom filters (like pre_get_posts etc)
 
+// Only show past events on event archive
+function event_archive_query($query) {
+  if ($query->is_post_type_archive('event') && is_archive() && !is_admin()) {
```

Comment:
Thats not what that condition is: https://codex.wordpress.org/Function_Reference/is_admin